docs/static/breaking-changes.asciidoc

```diff
@@ -209,10 +217,18 @@ bin/logstash-plugin install --version 4.0.1 logstash-output-kafka
 
 The version numbers were found by checking the compatibility matrix for the individual plugins.
 
-Specifically, for the Kafka input plugin, configs have changed a lot. Configuration `topic_id` 
-is replaced by `topics`. `zk_connect` is replaced by `bootstrap_servers` which are Kafka brokers. `consumer_id` is 
-replaced by `client_id`. Also, offsets are now stored in Kafka topics instead of Zookeeper. See <<plugins-inputs-kafka>> 
-for more information.
+[float]
+==== Kafka Input Configuration Changes
+
+As described in the section <<plugin-versions, above>>, Kafka plugin has been updated to bring in new consumer features. 
+In addition, to the plugin being incompatible with 0.8.x version of Kafka broker, _most_ of the config options have 
```

Comment:
Change to: "In addition to the plugin being incompatible with the 0.8.x version of the Kafka broker,